masterlist.yaml

```
@@ -829,6 +829,7 @@ plugins:
 
 ###### Locations - Overhauls ######
   ###Beautiful cities of Morrowind###
+   ##Core Files##
   - name: 'Beautiful cities of Morrowind.ESP'
```

I would the main subsection is enough fro the core files or none at all.
```suggestion
  ### Beautiful cities of Morrowind ###
  - name: 'Beautiful cities of Morrowind.ESP'
```